content/news/027/index.md

```diff
@@ -107,6 +107,32 @@ For more details, see the [changelog][tetra-changelog].
 
 ## Tooling Updates
 
+### [SPV]
+
+![SPV-0.1.0 screenshot](spv-0.1.0.png)
+
+[SPV] by [Albin Sjögren]
+is a calculator utility for working with astronomical position and velocity data.
+
+What was added for the first alpha release:
+
+- A new UI
+- Corrected vector normalizing
+- JSON and TXT exporting
+
+The primary features that are being worked on:
```

Comment:
Not sure if this is that important in this specific PR, but

> Avoid having multiple/nested bullet points.

Reply:
could remove "- Corrected vector normalizing" but i think the rest are essential

Reply:
I think given the lists are so short, this is probably fine - that guideline is mainly there to stop people from copying their entire changelog into the newsletter :)